libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -197,12 +197,19 @@ - (id)contentFromData:(NSData*)data fromResponse:(NSHTTPURLResponse*)response er
                 network.securityToken = securityToken;
             }
         }
+    } else if (!content && !responseError && self.statusCodeToRaiseErrorWithEmptyResponse.count > 0) {
```

Comment:
This is for cases where an app might want to treat certain response codes as errors, such as ```500``` in the above case, or ```304``` even (when ```etag``` headers are used and the content hasn't changed).